Workbooks/Azure Monitor - Workspaces/At Scale Insights/At Scale Insights.workbook

```diff
@@ -0,0 +1,453 @@
+                        "style": "decimal",
+                        "maximumFractionDigits": 2
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "Events Utilization Timeline",
+                    "formatter": 5
+                  }
+                ],
+                "rowLimit": 10000,
+                "filter": true,
+                "hierarchySettings": {
+                  "treeType": 1,
+                  "groupBy": [
+                    "Subscription"
+                  ],
+                  "expandTopLevel": true,
+                  "finalBy": "Name"
+                },
+                "labelSettings": [
+                  {
+                    "columnId": "Subscription",
+                    "label": "Subscription"
+                  },
+                  {
+                    "columnId": "Name",
+                    "label": "Active Time Series (Max)"
+                  },
+                  {
+                    "columnId": "microsoft.monitor/accounts--ActiveTimeSeries",
+                    "label": "Active Time Series"
+                  },
+                  {
+                    "columnId": "microsoft.monitor/accounts--ActiveTimeSeries Timeline",
+                    "label": "Active Time Series Timeline"
+                  },
+                  {
+                    "columnId": "Time Series Utilization",
+                    "label": "Time Series Utilization (%)"
+                  },
+                  {
+                    "columnId": "Time Series Utilization Timeline",
+                    "label": "Time Series Utilization Timeline"
+                  },
+                  {
+                    "columnId": "microsoft.monitor/accounts--EventsPerMinuteIngested",
+                    "label": "Events Per Minute Ingested"
+                  },
+                  {
+                    "columnId": "microsoft.monitor/accounts--EventsPerMinuteIngested Timeline",
+                    "label": "Events Per Minute Ingested Timeline"
+                  },
+                  {
+                    "columnId": "Events Utilization",
+                    "label": "Events Per Minute Utilization (%)"
+                  },
+                  {
+                    "columnId": "Events Utilization Timeline",
+                    "label": "Events Per Minute Utilization Timeline"
+                  }
+                ]
+              },
+              "sortBy": [],
+              "showExportToExcel": true
+            },
+            "showPin": true,
+            "name": "Azure Monitor workspace metrics",
+            "styleSettings": {
+              "margin": "0 10px 0 10px"
+            }
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "Subscription",
+        "comparison": "isNotEqualTo",
+        "value": ""
+      },
+      "name": "Storage accounts"
+    }
+  ],
+  "workbookPin": "storage account metrics",

```

Comment:
here this is saying "when pinned to a dashboard, pin a step with this name" but this step doesn't exist.  @ericajlin we should probably add a build validation for this?